### ISSUE TITLE
Does tensorflow support forward multiple times then backward once like average_loss in Caffe ?

### ISSUE THREAD

**Issue by YihangLou** [tensorflow/tensorflow#9878]
Hi, everyone. In my training tasks, the batch size is expected to be  large My training task is a little special that costs a lot of GPU memory. Now only set batch size = 6 almost run out of entire GPU memory 11GB GTX 1080Ti. However, I need to enlarge my batch size. In Caffe Platform, there is an **average_loss**  setting in solver.prototxt to perform forward multiple times and compute the average loss, then backward once. I scaned the documentation of tensorflow but there is no operation can implement such function. Does anyone have any ideas or how to use the existing op to implement this process?  Thanks

Following is my system info:
GTX1080Ti
CUDA version: 8.0
cuDNN:6.0
Tensorflow: r1.0.1
Ubuntu 16.04

**Comment by Carmezim** [comment]
Hu @YihangLou, this questions is better suited for StackOverflow. Github issues are only for bugs and feature requests.
Thanks.

**Comment by skye** [comment]
I'm gonna close this. @YihangLou, please open a new issue if you can't find a good solution via  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) and would like to request a new feature.
